### ISSUE TITLE
how to more times incremental learning

### ISSUE THREAD

**Issue by lvjs-ut** [JosephKJ/iOD#2]
Thank you for you good working! i try to work on my datasets 2cls + 5cls, and it's work!. But when i use ft after 2+5 to train more 4 cls, before 7 cls is not work  , ap = 0, so i want to know  how to work on （（2 + 5) + 4 ）+ 6 ......

**Comment by JosephKJ** [comment]
Hi @lvjs-ut : Glad to know that 2cls + 5cls works well! 

For any custom setting, you may have to adjust the hyper-parameters (learning rate, number of iterations for training, etc.) accordingly. 

Please reopen if you are still facing troubles.
